api_app/analyzers_manager/views.py

```diff
@@ -76,3 +78,19 @@ def perform_retry(self, report: AnalyzerReport):
         analyzers_controller.start_analyzers(
             report.job.id, analyzers_to_execute, runtime_configuration
         )
+
+
+class AnalyzerHealthCheckAPI(PluginHealthCheckAPI):
+    def get_cls_path(self, analyzer_name) -> str:
+        analyzer_dataclasses = AnalyzerConfigSerializer.get_as_dataclasses()
```

Comment:
The `get_as_dataclasses` function reads the entire config and then turns each `dict` object to a dataclass. So here there is much waste of computatation.

I think we could define a function, like:

```
@classmethod
AnalyzerConfig.get(cls, analyzer_name: str) -> Union[AnalyzerConfig, None]:
```

basically this would serve as queryset alike for dataclasses, to get an analyzer directly by it's name.